### DISCUSSION TITLE
Jetson Xavier AGX H01 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#401] [Q&A]
# Jetson Xavier AGX H01 | Seeed Studio Wiki

Jetson Xavier AGX H01

https://wiki.seeedstudio.com/Jetson_Xavier_AGX_H01_Driver_Installation/

**Comment by jinchenglee** [comment]
After flashing Jetpack 5.0.2 according to the steps on https://wiki.seeedstudio.com/Jetson_Xavier_AGX_H01_Driver_Installation/#tech-support--product-discussion, I noticed the fan is always on (probably at full speed).

However, when click “Run Jetson Power GUI” from pull-down menu of top-right corner “MODE 15W Desktop” app, the pane “Fan” shows:

Name fan1
Profile quiet
PWM 0.0%
RPM 0 rpm

It seems the system hasn’t read real fan information at all (definitely not 0 rpm, for example). I’m not sure how to fix this.

It is really noisy to have the fan always running. If I pull it off from the connected carrier board, I worry it might overheat, so it is a dilemma.

**Reply by MatthewJeffson** [reply]
Hi, just ask around and you might need to install jetson-stats:
```
sudo apt update
sudo apt install python3-pip -y
sudo pip3 install jetson-stats
```
You can find more information [here](https://wiki.seeedstudio.com/reComputer_Industrial_J20_Hardware_Interfaces_Usage/#monitor-system-performance) and control the fan through this application.

If I may, what kind of scenarios you are using with this high AI power devices :D
Regards,
Matthew

**Reply by jinchenglee** [reply]
It doesn't resolve the problem and I believe I've already installed jetson-stats before I tried to install it. 

jtop cannot read the pwm value, which shows 0 in the window. 

I just got a deal price for this as a 2nd hand. Haven't thought what to do, :)

**Comment by jinchenglee** [comment]
But the fan is connected to the seeedstudio carrier board, first of all. I'm using the Xavier AGX H01 dev kit, not the Nvidia AGX xavier dev kit. The latter has a different base board (carrier board) from the H01 one. 

The installation guide of the H01 dev kit (https://wiki.seeedstudio.com/Jetson_Xavier_AGX_H01_Driver_Installation/#comments), explicitly requires a different "board driver" to be copied before flashing. I suspect different mapping of the fan on the carrier board? 

Also, searching online shows a lot of fan control methods no longer works at all. Some directories are actually not there in the systerm (my system after flashing: Jetpack 5.0.2, L4T 35.1.0, the latest available to the H01 kit). I suspect Nvidia changed the way to control fan and the carrier board "driver" needs update too but it doesn't.

**Reply by MatthewJeffson** [reply]
Hi, I double check with my colleague. It seems like there are two versions of our carrier board on H01, which includes a 4-pin fan or a 2-pin fan. The 4-pin fan version H01 can be controlled by PWM using command, but the 2-pin fan version can only be controlled by plug in and plug out.
Would you mind check which version you have now?

**Reply by jinchenglee** [reply]
I do have a 2-pin fan. But it seems the socket it is plugged into have more pins but not connected? Is it possible to replace the fan to resolve this? Thanks.

**Reply by MatthewJeffson** [reply]
Yes, I think so. Would you mind check it and give the feedback to me.
Really appreciate.

**Reply by jinchenglee** [reply]
I did try to disassemble the cover of the heat sink - fan cover, but no luck. It seems the shell is clamped onto the heat sink, remove the screws won’t unlock the cover. In a word, I cannot access the fan to change it.

**Reply by MatthewJeffson** [reply]
Okay... That really needs to bring to the [tech support](https://wiki.seeedstudio.com/Jetson_Xavier_AGX_H01_Driver_Installation/#tech-support--product-discussion)

**Reply by MatthewJeffson** [reply]
"This is wrong guidance I said before"

**Reply by jinchenglee** [reply]
No, your guidance is wrong. Those two sticks seems to attach agx module shell top to the module’s pcb. It is too easy to ruin the whole agx compute module. Don’t play with that.

You should reattach the bottom cover of the module, then find the tool to open iPhone back (the sucker utilize air pressure) to pull the whole module out. The compute module is glued to the heat sink. 

I got access to the fan. However, it is 60x60mm 5v fan, which is very hard to find a 4pin pwm fan of 5v. They are usually 12v.

**Reply by MatthewJeffson** [reply]
Thank you very much for pointing it out! So glad I didn't do it before...

> I got access to the fan. However, it is 60x60mm 5v fan, which is very hard to find a 4pin pwm fan of 5v. They are usually 12v.

Okay maybe that's the reason for two versions... XD

It's been pleasure. Have a nice day!

**Comment by jinchenglee** [comment]
Can you upgrade the driver support to JP5.1.2?

**Reply by MatthewJeffson** [reply]
Hi, sorry but I asked around and the latest version for NVIDIA AGX Xavier H01 now is 5.1.1. 

[5.1.1.zip](https://github.com/Seeed-Studio/wiki-documents/files/12364988/5.1.1.zip)

**Reply by tomtana** [reply]
are you sure this is the right one for Jetson Xavier AGX H01 ? in the readme there is jetson-agx-orin-devkit mentioned..

**Comment by jjannik** [comment]
Where is the Debug UART Port for this device ?

**Reply by xzxcessarr** [reply]
Hello, here is my answer: https://github.com/Seeed-Studio/wiki-documents/discussions/474#discussioncomment-6861620

**Comment by ghost** [comment]
I'm seeing the following error:
> CPU Bootloader is not running on device.

I am using the Seeed H01 board, with the 5.0.2 driver package and matching R35.1.0 OS. Host machine is a 18.04 Ubuntu VM.
Followed the steps from https://wiki.seeedstudio.com/Jetson_Xavier_AGX_H01_Driver_Installation/ , including download links. The issue pops up at this command, in the readme of the drivers package:
``` sudo ./flash.sh jetson-xavier mmcblk0p1 ```

The output of the command above:
https://pastebin.com/DDKpYEBM

Any idea of what the issue could be, or things worth trying?

**Reply by xzxcessarr** [reply]
Hello, after reading your description we recommend you to re-flash the official jetpack without drivers, if there is still have error please inform us

**Reply by tomtana** [reply]
I do experience exactly the same issue. Tried a couple of different versions 5.* 4.* with and without custom drivers. any suggestion how to fix this?

**Reply by Lofty34** [reply]
To flash without H01 Drivers, Do you just skip the step (cp -a -f H01_Driver/Linux_for_Tegra/kernel Linux_for_Tegra/) and run the flash command?

**Reply by Ainmimdal** [reply]
any solutions?

**Comment by fahriwps** [comment]
Hi, I have this exact Jetson AGX Xavier that cannot power on. Both power adapters (AGX Xavier and AGX Orin) have the same specifications and work normally with my Jetson AGX Orin. However, when either adapter is connected to the AGX Xavier:

The adapter LED turns off

A repeating beep sound occurs

The board does not power on

The adapters work normally again after unplugging from the Xavier.

This behavior is consistent with both adapters. Is this likely a short circuit or power input failure on the AGX Xavier?

**Reply by bleaaach** [reply]
Hi, based on the symptoms you described, this is likely a **short circuit or severe overcurrent** on the **19V power input** of the Xavier board.

If you have a multimeter, you can perform the following test **with the power completely disconnected**:

### Steps:
1. Disconnect all power supplies.
2. Set the multimeter to **resistance mode (Ω)**.
3. Measure between:
   - **19V power input positive**
   - **Ground (GND)**

### Judgment:
- **Normal**: several thousand ohms (kΩ) or higher
- **Short circuit**: close to **0 Ω** or below **5 Ω**

If the reading is close to 0 Ω, a power input short circuit is confirmed.
If this is the case, please send an email to **techsupport@seeed.io**. We will arrange a replacement for you if the board is still under warranty.
